From: Egbert Eich <eich@suse.de>

Make error message about FBIOBLANK missing less prominent but more telling

The FBIOBLANK ioctl is not present in some fbdev kernel drivers. This 
however is not a huge problem. This don't mark it as error in the log
instead give more information on the possible cause.

Index: hw/xfree86/fbdevhw/fbdevhw.c
===================================================================
--- hw/xfree86/fbdevhw/fbdevhw.c.orig
+++ hw/xfree86/fbdevhw/fbdevhw.c
@@ -858,9 +858,10 @@ fbdevHWDPMSSet(ScrnInfoPtr pScrn, int mo
         return;
     }
 
+    /* Novell Bug #146462 */
     if (-1 == ioctl(fPtr->fd, FBIOBLANK, (void *) fbmode))
-        xf86DrvMsg(pScrn->scrnIndex, X_ERROR,
-                   "FBIOBLANK: %s\n", strerror(errno));
+        xf86DrvMsg(pScrn->scrnIndex, X_INFO,
+                   "FBIOBLANK: %s (Screen blanking not supported by vesafb of Linux Kernel)\n", strerror(errno));
 }
 
 Bool
@@ -875,9 +876,10 @@ fbdevHWSaveScreen(ScreenPtr pScreen, int
 
     unblank = xf86IsUnblank(mode);
 
+    /* Novell Bug #146462 */
     if (-1 == ioctl(fPtr->fd, FBIOBLANK, (void *) (1 - unblank))) {
-        xf86DrvMsg(pScrn->scrnIndex, X_ERROR,
-                   "FBIOBLANK: %s\n", strerror(errno));
+        xf86DrvMsg(pScrn->scrnIndex, X_INFO,
+                   "FBIOBLANK: %s (Screen blanking not supported by vesafb of Linux Kernel)\n", strerror(errno));
         return FALSE;
     }
 
